book/getting-started/linux-setup.md

```diff
@@ -37,15 +37,15 @@ sudo apt-get install -y build-essential
 sudo apt-get install -y libssl-dev
 ```
 
-### Install GCC 5 or Higher
+### Install GCC 4.7 or Higher
 
-You'll need to be using at least `gcc-5`. We rely on its atomics support. If you have at least `gcc-5` installed on your machine, you don't need to do anything. If you have gcc 4 or lower, you'll need to upgrade. You can check your `gcc` version by running:
+ If you have at least GCC 4.7 installed on your machine, you don't need to do anything. If you have gcc 4.6 or lower, you'll need to upgrade. You can check your `gcc` version by running:
 
 ```bash
 gcc --version
 ```
 
-To upgrade:
+To upgrade to GCC 5 if you don't have at least GCC 4.7:
```

Comment:
I could see this being a little confusing because my expectation would be that if I need at least version X then I'll get instructions on how to install version X, but this is telling me to install version Y. I could see someone wondering why they're being told to install version 5 when we just told them that they only need version 4.7.

The clearer approach would be to tell them how to install 4.7.

If I'm the only one who thinks this is potentially confusing then I'll shut up.

Reply:
Meh, I'm not going to hold things up because of this.